model/shift_design_objective.py

```diff
@@ -2,17 +2,24 @@
 
 
 class ShiftDesignObjective:
-    def __init__(self, model, var, weights, shifts, time_periods):
+    def __init__(self, model, var, weights, shifts, time_periods, low_dur_shifts, long_dur_shifts):
         self.model = model
         self.shifts = shifts
         self.time_periods = time_periods
+        self.low_dur_shifts = low_dur_shifts
+        self.long_dur_shifts = long_dur_shifts
 
-        self.add_objective(weights, var.y, var.delta)
+        self.add_objective(weights, var.y, var.delta, var.rho)
 
-    def add_objective(self, weights, y, delta):
+    def add_objective(self, weights, y, delta, rho):
         self.model.setObjective(
-            quicksum(y[t_marked, v] for t_marked, v in self.shifts)
+            quicksum(y[t_1, v_1] for t_1, v_1 in self.shifts)
```

Comment:
hvorfor ikke bare `t, v in self.shifts`?

Reply:
Dette finner jeg ikke igjen i egen kode. I egen kode står det at t_1, t_2 og t_3 benyttes i add_objective.